paper/paper.md

```diff
@@ -26,18 +29,60 @@ affiliations:
     index: 3
   - name: Division of Epidemiology and Biostatistics, School of Public Health, University of California, Berkeley
```

Comment:
```suggestion
  - name: Division of Biostatistics, School of Public Health, University of California, Berkeley
```
I think the Biostatistics and Epidemiology Divisions have become independent after the merger a year or two ago.